src/sdkRuntimeModels.ts

```diff
@@ -220,12 +224,19 @@ export interface SDKIdentityApi {
 export interface SDKHelpersApi {
     createServiceUrl(arg0: string, arg1: string): void;
     createXHR?(cb: () => void): XMLHttpRequest;
-    extend(...args: any[]);
-    parseNumber(value: string | number): number;
+    extend?(...args: any[]);
+    parseNumber?(value: string | number): number;
```

Comment:
It's because `BatchValidator`'s `getMPInstance` method returns the entire mPInstance, and since the `HelperAPI` is tightly coupled, we either need to define every single helper method in the `getMPInstance` return function, or we can make them optional here and TS will ignore them.

Reply:
For sustained engineering: https://go.mparticle.com/work/SQDSDKS-5053

Reply:
got it :(